From b698779053b7ba31bd8e69b230e86515e3019bf6 Mon Sep 17 00:00:00 2001
From: K900 <me@0upti.me>
Date: Sun, 5 Nov 2023 22:04:32 +0300
Subject: [PATCH] Force add the registry entry

Otherwise, newer Wine versions will prompt to overwrite it and loop there forever.
---
 src/backend/storeManagers/legendary/setup.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/backend/storeManagers/legendary/setup.ts b/src/backend/storeManagers/legendary/setup.ts
index 1837106621..b5c2432435 100644
--- a/src/backend/storeManagers/legendary/setup.ts
+++ b/src/backend/storeManagers/legendary/setup.ts
@@ -20,7 +20,7 @@ export const legendarySetup = async (appName: string) => {
 
   // Fixes games like Fallout New Vegas and Dishonored: Death of the Outsider
   await runWineCommandOnGame(appName, {
-    commandParts: ['reg', 'add', 'HKEY_CLASSES_ROOT\\com.epicgames.launcher'],
+    commandParts: ['reg', 'add', 'HKEY_CLASSES_ROOT\\com.epicgames.launcher', '/f'],
     wait: true,
     protonVerb: 'waitforexitandrun'
   })
